src/test/java/org/jabref/gui/duplicationFinder/DuplicateSearchTest.java

```diff
@@ -0,0 +1,109 @@
+package org.jabref.gui.duplicationFinder;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.Optional;
+
+import org.jabref.gui.DialogService;
+import org.jabref.gui.Globals;
+import org.jabref.gui.JabRefFrame;
+import org.jabref.gui.LibraryTab;
+import org.jabref.gui.StateManager;
+import org.jabref.gui.undo.CountingUndoManager;
+import org.jabref.gui.undo.NamedCompound;
+import org.jabref.gui.util.OptionalObjectProperty;
+import org.jabref.logic.l10n.Localization;
+import org.jabref.model.database.BibDatabase;
+import org.jabref.model.database.BibDatabaseContext;
+import org.jabref.model.database.BibDatabaseMode;
+import org.jabref.model.entry.BibEntry;
+import org.jabref.model.entry.field.StandardField;
+import org.jabref.model.entry.types.StandardEntryType;
+import org.jabref.preferences.JabRefPreferences;
+import org.jabref.testutils.category.GUITest;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.testfx.framework.junit5.ApplicationExtension;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.spy;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@GUITest
+@ExtendWith(ApplicationExtension.class)
+public class DuplicateSearchTest {
+
+    private final DialogService dialogService = spy(DialogService.class);
+    private final StateManager stateManager = mock(StateManager.class);
+    private final JabRefFrame jabRefFrame = mock(JabRefFrame.class);
+    private final LibraryTab libraryTab = mock(LibraryTab.class);
+    private final BibDatabaseContext bibDatabaseContext = mock(BibDatabaseContext.class);
+    private final CountingUndoManager undoManager = mock(CountingUndoManager.class);
+
+    private DuplicateSearch duplicateSearch;
+    private BibEntry entry1;
+
+    @BeforeEach
+    void setup() {
+        if (Globals.prefs == null) {
+            Globals.prefs = JabRefPreferences.getInstance();
+        }
```

Comment:
This should go into `@BeforeAll`, shouldn't it? Then, it can be just the assignment of the `Globals.prefs` variable.

Moreover, the preferences need to be mocked to narrow down the impact of the preferences. A test should never rely on the users's configuration.